nixos/tests/hedgedoc.nix

```diff
@@ -8,20 +8,22 @@ import ./make-test-python.nix ({ pkgs, lib, ... }:
 
   nodes = {
     hedgedocSqlite = { ... }: {
-      services = {
-        hedgedoc = {
-          enable = true;
-          settings.dbURL = "sqlite:///var/lib/hedgedoc/hedgedoc.db";
-        };
-      };
+      services.hedgedoc.enable = true;
     };
 
     hedgedocPostgres = { ... }: {
       systemd.services.hedgedoc.after = [ "postgresql.service" ];
       services = {
         hedgedoc = {
           enable = true;
-          settings.dbURL = "postgres://hedgedoc:\${DB_PASSWORD}@localhost:5432/hedgedocdb";
+          settings.db = {
+            dialect = "postgres";
+            user = "hedgedoc";
+            password = "\${DB_PASSWORD}";
+            host = "localhost";
+            port = 5432;
```

Comment:
We want to use socket connection

Reply:
Could we do one of both? This test saved me from a bug I accidentally created while working on the module that would only appear for the localhost tcp socket.

Reply:
I've split it into two.